src/api/app/controllers/webui/package_controller.rb

```diff
@@ -786,6 +786,7 @@ def live_build_log
 
     if Project.exists_by_name(params[:project])
       @project = Project.get_by_name(params[:project])
+      @project = @project.name if @project.kind_of?(Project)
```

Comment:
On Freitag, 18. M�rz 2016, 07:07:11 wrote Henne Vogelsang:
> > @@ -786,6 +786,7 @@ def live_build_log
> >  
> >      if Project.exists_by_name(params[:project])
> >        @project = Project.get_by_name(params[:project])
> > +      @project = @project.name if @project.kind_of?(Project)
> 
> Why do you make a string out of it? Everyone will expect `@project` to be an object.

But you need to check then in the views what kind of object it is
and handle remote and local different there.

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany